src/krishna/Test2_Krishna_P1.java

```diff
@@ -0,0 +1,51 @@
+package krishna;
+
+public class Test2_Krishna_P1 {
+
+	int sum = 6;
+
+	void find_ArrayPairs(int arr[], int n) {
+
+		for (int i = 0; i < n; i++)
+			for (int j = i + 1; j < n; j++)
+				if (arr[i] + arr[j] == sum)
+					System.out.print(" {" + arr[i] + " , " + arr[j] + "},");
+	}
+
+	void findSingleoccurance(int arr1[], int m) {
+		int count = 0;
+		int temp = 0;
+		int i = 0;
+		for (; i < m; i++) {
+			int j = 0;
+			for (; j < m; j++) {
+				if (i == j) {
```

Comment:
I would recommend to check below code, small change requires.
https://github.com/KrishnaKTechnocredits/JAVATechnoDec2021/pull/845/files